us/en/skymiles/scripts/scripts.js

```diff
@@ -344,6 +359,14 @@ async function loadLazy(doc) {
   sampleRUM('lazy');
   sampleRUM.observe(main.querySelectorAll('div[data-block-name]'));
   sampleRUM.observe(main.querySelectorAll('picture > img'));
+
+  Object.keys(hreflangMap).forEach((key, value) => {
+    const link = document.createElement('link');
+    link.setAttribute('rel', 'alternate');
+    link.setAttribute('hreflang', key);
+    link.setAttribute('href', window.location.href.replace(/https:\/\/(www\.)?delta\.com\/us\/en/, value));
```

Comment:
Looks like this doesn't work well in dev and on the hlx pages.
Maybe we need to change this to something like:
```js
link.setAttribute('href', `${value}${window.location.pathname.replace(/\/us\/en/, '')}`);
```